js/js_primitives9.js

```diff
@@ -0,0 +1,8 @@
+// 9.) Change the word "Space" to the word "Spice".
+
+var word = "Space";
+
+let spaceArr = word.split('');
+spaceArr.splice(spaceArr.indexOf('a'),1,'i');
+let spaceArrStr = spaceArr.join('');
+console.log(spaceArrStr);
```

Comment:
Good answer - recommend revisiting and solving with the built in string function .replace()